oktapam/resource_sudo_command_bundle_test.go

```diff
@@ -0,0 +1,57 @@
+package oktapam
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/okta/terraform-provider-oktapam/oktapam/client"
+	"github.com/okta/terraform-provider-oktapam/oktapam/constants/attributes"
+	"github.com/okta/terraform-provider-oktapam/oktapam/logging"
+	"github.com/okta/terraform-provider-oktapam/oktapam/utils"
+)
+
+func TestAccResourceSudoCommandBundle(t *testing.T) {
```

Comment:
can we add update test?
Also if not already, can we modify the policy terraform to include adding/updating rules with sudo.

Reply:
Sure I can look into it.